styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -566,6 +582,8 @@ private void emitPodEvents(Watcher.Action action, Pod pod, RunState runState) {
       }
 
       try {
+        // TODO: spoofing counter values like this can give unexpected results, e.g. if we emit two events here the
+        // first one might be discarded and the second one accepted.
```

Comment:
We can continue with https://github.com/spotify/styx/pull/445 if we see high risk here.